main/src/flix/test/TestInterface.java

```diff
@@ -0,0 +1,4 @@
+package test.flix;
+
+public interface TestInterface {
```

Comment:
We have had some trouble with tests depending on Java sources.

Perhaps you could simply use:

- java.lang.Object
- java.io.Serializable
- java.lang.Cloneable

Reply:
Can do, but I can see that there are tests in, for example, `flix/main/test/flix/Test.Exp.Jvm.GetField.flix` which depend upon `test.flix.TestClass`? Can you explain the "boundaries" of the problems?

Reply:
> Can do, but I can see that there are tests in, for example, `flix/main/test/flix/Test.Exp.Jvm.GetField.flix` which depend upon `test.flix.TestClass`? Can you explain the "boundaries" of the problems?

Its possible it has been resolved :)

Reply:
The problem is/was how Java source code is compiled and made available to the Flix test suite.